src/ws/cockpitcertificate.c

```diff
@@ -220,6 +220,8 @@ sscg_make_dummy_cert (const gchar *key_file,
     "--ca-file", ca_file,
     "--hostname", cn,
     "--organization", org,
+    "--subject-alt-name", "localhost",
+    "--subject-alt-name", "127.0.0.1",
```

Comment:
This will need to be changed to:
`"--subject-alt-name", "IP:127.0.0.1/255.255.255.255",`
which will work properly for SSCG 2.3.0 and later.

If the system has 2.2.0 or earlier, the net effect will be that there will be a nonsensical value of `DNS:IP:127.0.0.1/255.255.255.255` in the certificate which will just never be usable for validating anything.

Reply:
Why the /255..? Usually one writes just `IP:127.0.0.1` if it's just for a single host without a mask/pattern.